djangobaseview/static/js/guestbook/view/GreetingView.js

```diff
@@ -0,0 +1,80 @@
+define([
+	"dojo/_base/declare",
+	"dojo/_base/lang",
+	"dojo/dom",
+	"dojo/on",
+	"dojo/dom-style",
+	"dojo/topic",
+	"dojo/text!../templates/GreetingView.html",
+	"./_ViewBaseMixin",
+	"../store/GreetingStore"
+], function (declare, lang, dom, on, domStyle, topic, template, _ViewBaseMixin, GreetingStore) {
+
+	return declare("guestbook.view.GreetingView", [_ViewBaseMixin], {
+		templateString: template,
+
+		constructor: function(guestbook) {
+			this.greetingId = guestbook.greeting_id;
+			this.createDate = guestbook.date;
+			this.guestbookName = guestbook.guestbook_name;
+			this.content = guestbook.content;
+			this.updatedBy = guestbook.updated_by;
+			this.updatedDate = guestbook.updated_date;
+		},
+
+		_deleteGreeting: function () {
+			var confirm = window.confirm("Do you want delete " + this.guestbookName);
+			var destroyGreeting = lang.hitch(this, this.destroy());
+
+			if (confirm == true) {
+				var store = new GreetingStore(this.guestbookName, this.content, this.greetingId);
+
+				store.deleteGreeting().then(function() {
```

Comment:
chỗ này em đã thử dùng và nó hiểu là cái this nào đó anh ko phải this của template